dask/bytes/s3.py

```diff
@@ -13,15 +14,46 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_s3(storage_options=None):
```

Comment:
Generally I find functions with explicit argument names to be more clear.  It also cleans up code like `if '_username' in storage_options` to just `if username:`